src/Tools/AI Test Toolkit/src/AITTestRunIteration.Codeunit.al

```diff
@@ -144,6 +162,14 @@ codeunit 149042 "AIT Test Run Iteration"
 
         // Update AI Token Consumption
         GlobalAITokenUsedByLastTestMethodLine := 0;
+
+        // Update Turns
+        GlobalNumberOfTurnsPassedForLastTestMethodLine := 0;
+        GlobalNumberOfTurnsForLastTestMethodLine := 1;
+
+        // Update Test Accuracy
+        GlobalTestAccuracy := 0.0;
```

Comment:
Minor - You do not need to write 0.0. Same for 1.0. It will convert if you simply write 0.